apps/website/src/data/collaborations.ts

```diff
@@ -428,12 +428,12 @@ const collaborations: Collaboration[] = (
         {
           name: "PeachJars",
           image: peachJars,
-          popularity: 720_000, // Twitter followers, 2024-04-07
+          popularity: 744_107, // Twitter followers, 2024-10-4
```

Comment:
```suggestion
          popularity: 744_00, // Twitter followers, 2024-10-4
```
Rounded

Reply:
@MattIPv4 noticed it now. The rounded-up value must be 744_000 or did you mean to go by that number?
Let me know and I will append the zero.

Reply:
I definitely missed a zero, good spot

Reply:
updated the same. please have a look and let me know if any other changes could be of use